L1Trigger/DemonstratorTools/test/gtt/createFirmwareInputFiles_cfg.py

```diff
@@ -104,11 +104,12 @@
 process.l1tTrackerEmuHTMiss.debug = (options.debug > 0)
 
 #Disable internal track selection
+#There is a problem with setting all of these (especially eta) to high numbers.
 process.l1tTrackJetsEmulation.MaxDzTrackPV = cms.double(10000.0)
 process.l1tTrackJetsEmulation.trk_zMax = cms.double(10000.0)    # maximum track z
 process.l1tTrackJetsEmulation.trk_ptMax = cms.double(10000.0)    # maximumum track pT before saturation [GeV]
```

Comment:
I think this (trk_ptMax) and the line below (trk_ptMin) should be removed from the configuration, we shouldn't touch them according to George.

Reply:
Ok, I guess I will leave trk_ptMin and trk_ptMax in for now, but we can try to find out if we really need them at all?

Reply:
I'd just remove them from the createFirmwareInputFiles_cfg.py, unless you know that the ptMax needs to be different to match any saturation in firmware (this is a concern...)